debian/control

```diff
@@ -1,13 +1,13 @@
-Source: joyent-mdata-client
-Maintainer: Joshua M. Clulow <jmc@joyent.com>
+Source: triton-mdata-client
+Maintainer: Triton Developers <triton-devs@mnx.io>
 Section: misc
 Priority: optional
 Standards-Version: 3.9.4
 Build-Depends: debhelper (>= 8)
 
-Package: joyent-mdata-client
+Package: triton-mdata-client
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Homepage: https://github.com/joyent/mdata-client
+Homepage: https://github.com/TritonDataCenter/mdata-client
 Description: Metadata client tools for use in guest virtual machines
```

Comment:
It's probably a good idea to add `Provides: joyent-mdata-client` and `Conflicts: joyent-mdata-client` so that dpkg will know that this is the successor. See `deb-control(5)` on a debian system for details.

Reply:
Nice, this is a really important detail. Thank you for catching this.